src/main/java/Task.java

```diff
@@ -27,7 +40,7 @@ protected static int getSize() {
         return size;
     }
 
-    protected static Task generateTask(String input) throws Exception {
+    public static Task generateTask(String input) throws Exception {
```

Comment:
Just a genuine question but why did you change protected to public? Thanks!!

Reply:
I packaged `Task` and its subclasses under `duke.task`. So for an outsider class to be able to call `generateTask` the `public`  access modifier is needed. Do note that in the absence of a modifier it is default to package private.